salt/kubelet/node_labels.jinja

```diff
@@ -0,0 +1 @@
+KUBELET_NODE_LABELS="--node-labels={{ node_labels|join(",") }}"
```

Comment:
That will not work. `/etc/kubernetes/kubelet` cannot source other files. It is not interpreted by a shell. You also cannot do parameter expansion.

https://ask.fedoraproject.org/en/question/39528/systemd-when-using-environmentfile-variables-are-not-expanded/

So, we have to rewrite the entire systemd unit. I'll rename things to -initial though.